src/Microsoft.Azure.EventHubs.Processor/PartitionManager.cs

```diff
@@ -221,9 +222,9 @@ async Task RunAsync()
                         {
                             ourLeaseCount++;
                             ProcessorEventSource.Log.PartitionPumpInfo(this.host.HostName, lease.PartitionId, "Trying to renew lease.");
-                            renewLeaseTasks.Add(leaseManager.RenewLeaseAsync(lease).ContinueWith(renewResult =>
+                            renewLeaseTasks.Add(leaseManager.RenewLeaseAsync(lease).ContinueWith(async renewResult =>
```

Comment:
>async renewResult [](start = 97, length = 17)

This will break the wait chain at below because returned task will be a Task<Task> type. 

await Task.WhenAll(renewLeaseTasks).ConfigureAwait(false);